src/redist/targets/BundledSdks.targets

```diff
@@ -11,5 +11,6 @@
     <BundledSdk Include="FSharp.NET.Sdk" Version="1.0.4-bundled-0100" />
     <BundledSdk Include="Microsoft.Docker.Sdk" Version="1.1.0" />
     <BundledSdk Include="ILLink.Tasks" Version="$(ILLinkTasksPackageVersion)" />
+    <BundledSdk Include="Microsoft.DotNet.Build.Bundle" Version="$(MicrosoftDotnetBuildBundlePackageVersion)" />
```

Comment:
Seeing this next to the other Microsoft.NET.* has me thinking we should strong consider Microsoft.NET here instead of Microsoft.DotNet.

Reply:
@nguerrera I named the package as `Microsoft.DotNet.Build.Bundle` similar to `Microsoft.DotNet.PlatformAbstractions` in the [`core-setup` repo](https://github.com/dotnet/core-setup/tree/master/src/managed). 

I'm happy to change the name to `Microsoft.NET.Build.Bundle` if you think that's better.
CC: @KathleenDollard  for package naming convention.

Reply:
@terrajobst can you please comment about this issue about package naming convention? Thanks.

Reply:
Just to be clear, this name isn't really customer facing, but it's just that I think the folder name should match its siblings.

We don't have to block the initial insertion on this.

Reply:
I don't think we ever created an official guideline. Let me start a mail thread so we close on this.

Reply:
Thanks @terrajobst

Reply:
>> We don't have to block the initial insertion on this.
Thanks @nguerrera. I agree. But we're blocked on the failing test leg nevertheless.